control/control_test.go

```diff
@@ -77,7 +77,7 @@ func (m *MockPluginManagerBadSwap) teardown()                         {}
 func (m *MockPluginManagerBadSwap) SetPluginConfig(*pluginConfig)     {}
 func (m *MockPluginManagerBadSwap) SetMetricCatalog(catalogsMetrics)  {}
 func (m *MockPluginManagerBadSwap) SetEmitter(gomit.Emitter)          {}
-func (m *MockPluginManagerBadSwap) GenerateArgs(string) plugin.Arg    { return plugin.Arg{} }
+func (m *MockPluginManagerBadSwap) GenerateArgs(int) plugin.Arg       { return plugin.Arg{} }
```

Comment:
It's to implement the `managesPlugins` interface defined inside control/control.go at line 128.